src/ggrc/assets/javascripts/apps/business_objects.js

```diff
@@ -39,6 +39,7 @@
         process: CMS.Models.Process,
         control: CMS.Models.Control,
         assessment: CMS.Models.Assessment,
+        assessment_template: CMS.Models.AssessmentTemplate,
```

Comment:
Just  checking - is this needed? Because on the backend, the `AssessmentTemplate` class does not inherit from the `BusinessObject` mixin.